specification/devopsinfrastructure/resource-manager/Microsoft.DevOpsInfrastructure/preview/2024-04-04-preview/devopsinfrastructure.json

```diff
@@ -1419,19 +1393,30 @@
           "type": "integer",
           "format": "int64",
           "description": "The maximum permitted usage of the resource."
-        },
-        "name": {
-          "$ref": "#/definitions/QuotaName",
-          "description": "The details of the quota."
         }
       },
       "required": [
+        "name",
```

Comment:
Please take a look at the optionality (with `?`) if you don't want the breaking change to flag these.